site/src/sphinx/advanced-structured-logging.rst

```diff
@@ -8,12 +8,12 @@ retrieving the information collected during request life cycle in a machine-frie
 
 What properties can be retrieved?
 ---------------------------------
-:api:`RequestLog` provides all the properties you can retrieve:
+:api:`RequestLog` provides the properties you can retrieve:
 
 +----------------------------------------------------------------------------------------------------+
 | Request properties                                                                                 |
 +=============================+======================================================================+
-| ``requestStartTimeMillis``  | when the request processing started                                  |
+| ``requestStartTimeMicros``  | when the request processing started                                  |
```

Comment:
Could you add that this value is the number of microseconds since the epoch (01-Jan-1970 00:00:00 UTC)?